easycv/datasets/ocr/pipelines/transform.py

```diff
@@ -0,0 +1,1274 @@
+# Modified from https://github.com/PaddlePaddle/PaddleOCR/tree/release/2.6/ppocr/data/imaug
+import math
+import random
+import sys
+
+import cv2
+import imgaug
+import imgaug.augmenters as iaa
+import numpy as np
+import pyclipper
+from shapely.geometry import Polygon
+
+from easycv.datasets.registry import PIPELINES
+
+
```

Comment:
file too long, split into smaller files according to function

Reply:
done